ui/src/components/view/TreeView.vue

```diff
@@ -213,6 +215,40 @@ export default {
       }
 
       this.reloadTreeData(newData)
+    },
+    treeVerticalData () {
+      if (!this.domainStore.isExpand) {
+        return
+      }
+      if (this.domainStore.expands && this.domainStore.expands.length > 0) {
+        for (let i = 0; i < this.domainStore.expands.length; i++) {
```

Comment:
Suggestion:
I think it will be more interesting if we use a `for...of` here, to simplify the code.

```js
for (let expandKey of this.domainStore.expands) {
...
```